Easydict/Feature/ViewController/Window/BaseQueryWindow/EZBaseQueryViewController.m

```diff
@@ -245,13 +244,31 @@ - (void)dealloc {
 - (void)handleServiceUpdate:(NSNotification *)notification {
     NSDictionary *userInfo = notification.userInfo;
     EZWindowType type = [userInfo[EZWindowTypeKey] integerValue];
+    NSString *serviceType = [notification.userInfo objectForKey:EZServiceTypeKey];
+    if ([serviceType length] != 0 && type == self.windowType) {
+        [self updateSingleService:serviceType windowType:type];
+        return;
+    }
     if (type == self.windowType || !userInfo) {
-        [self updateServices];
+        [self updateServices:[EZLocalStorage.shared allServices:self.windowType]];
+    }
+}
+
+- (void)updateSingleService:(NSString *)serviceType windowType:(EZWindowType)windowType {
```

Comment:
This method looks good, but the actual effect may not be good, because the three query windows are currently independent, and they all share these services, so all window services usually need to be updated.

Reply:
use `self.windowType` instead.